### ISSUE TITLE
Cannot read property 'priority' of undefined

### ISSUE THREAD

**Issue by pajtai** [vuejs/vue-touch#65]
After updating to the latest vue (2.1.8) getting the following error using vue-touch v1.1.0

Cannot read property 'priority' of undefined

It is caused by:

```
 Vue.directive('touch', {

      isFn: true,
      acceptStatement: true,
      priority: Vue.directive('on').priority,
```

Looks like `Vue.directive('on')` now returns `undefined`. Assume that it previously returned an object.

**Comment by LinusBorg** [comment]
@pajtai The current version of vue-touch is only compatible with Vue 1.*

I just released vue-touch 2.0.0-beta.1 though, which works with Vue 2.* 

https://github.com/vuejs/vue-touch/releases/tag/2.0.0-beta.1

**Comment by Stevendwy** [comment]
npm install vue-touch@next
